src/setupProxy.js

```diff
@@ -11,12 +13,23 @@ const archivePaths = [
   'specials',
 ];
 
-const ARCHIVE_URL = process.env.ARCHIVE_URL;
 
 module.exports = function(app) {
+  process.env.NODE_ENV === 'test' || process.env.CI
```

Comment:
hmm, I'm only familiar with `production` and `development` as being valid values for `NODE_ENV` ... is `test` another commonly-used value?

Reply:
its what create-react-app sets it to in testing https://github.com/facebook/create-react-app/blob/master/packages/react-scripts/scripts/test.js#L13